br/tests/lightning_routes/run.sh

```diff
@@ -4,12 +4,17 @@
 
 set -eux
 
+echo "testing checksum-error..."
+export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/importer/checksum-error=1*return(true)"
```

Comment:
```suggestion
export GO_FAILPOINTS="github.com/pingcap/tidb/br/pkg/lightning/importer/checksum-error=1*return()"
```

I remember this is OK